tests/run_tests.py

```diff
@@ -265,6 +265,8 @@ def addToTestListIfMatch(someTest):
 addToTestListIfMatch(Test.wasmOnly('unit/threading/atomic3.cpp', [['-pthread'], ['-pthread','-cheerp-make-module=es6'], ['-pthread','-cheerp-make-module=commonjs'], ['-pthread', '-cheerp-make-module=closure']]))
 addToTestListIfMatch(Test.wasmOnly('unit/threading/atomic4.cpp', [['-pthread'], ['-pthread','-cheerp-make-module=es6'], ['-pthread','-cheerp-make-module=commonjs'], ['-pthread', '-cheerp-make-module=closure']]))
 addToTestListIfMatch(Test.wasmOnly('unit/threading/thread_setup.cpp', [['-pthread'], ['-pthread','-cheerp-make-module=es6'], ['-pthread','-cheerp-make-module=commonjs'], ['-pthread', '-cheerp-make-module=closure']]))
+addToTestListIfMatch(Test.common('unit/exceptions/exitexception1.cpp', [['-fexceptions','-cheerp-make-module=commonjs']]))
```

Comment:
you forgot to add the es6 module version?

Reply:
I did indeed forget to add the es6 versions to the commit.